### PR TITLE
Note measuring conversions in the ad guide

### PR DESCRIPTION
**Pull request by davidfischer**

### CODE REVIEW

**Comment by davidfischer** [comment]
Should we add a little thing about adding the `${advertisement}` or `${publisher}` parameters?

**Comment by ericholscher** [comment]
> Should we add a little thing about adding the `${advertisement}` or `${publisher}` parameters?

Yea, seems useful 👍

File: content/pages/learning-hub/advertiser-guide.md [hunk 1]
```diff
@@ -96,6 +96,19 @@ In terms of high-performing ad images we have a few recommendations:
 Our [SaaS product example](#saas-product) example shows a good combination of this approach,
 but it's worth trying more focused variations as well.
 
+
+## Measuring conversions
+
+Rather than using pixels or beacons which are frequently blocked by ad or tracker blockers,
+we recommend adding UTM or other parameters to your landing page links.
```
Comment (ericholscher) [inline content/pages/learning-hub/advertiser-guide.md:103]:
```suggestion
we recommend adding UTM parameters to your landing page links.
```

The `other` here is weird without explaining it.

File: content/pages/learning-hub/advertiser-guide.md [hunk 2]
```diff
@@ -96,6 +96,19 @@ In terms of high-performing ad images we have a few recommendations:
...
+This keeps your measurements simple and lets you easily compare the actual conversions
+you're seeing from us against any other network where your ads are running.
+We are confident in our performance and think you'll see the results speak for themselves.
```
Comment (ericholscher) [inline content/pages/learning-hub/advertiser-guide.md:106]:
```suggestion
This keeps your measurements simple and lets you easily compare the conversions
you're seeing from us against other networks you're using.
We are confident in our performance and know the results will speak for themselves.
```

`think you'll see` is awkward. Tried to rephrase.

File: content/pages/learning-hub/advertiser-guide.md [hunk 3]
```diff
@@ -96,6 +96,19 @@ In terms of high-performing ad images we have a few recommendations:
...
+
+Our advertiser platform will measure all performance metrics before a user clicks-through
+to your landing page like the number of impressions and click-through rate your ads receive.
```
Comment (ericholscher) [inline content/pages/learning-hub/advertiser-guide.md:109]:
```suggestion
to your landing page like the number of impressions and click-through rate (CTR) your ads receive.
```

**Review by ericholscher — APPROVED** [review]
This is great. I think we could expand on UTM codes with an example, and think the magic variables should likely also be mentioned here. Could we useful to have an example at the bottom like:

`https://example.com?utm_source=ethicalads&utm_campaign=${advertisement}&utm_content=${publisher}` for example?

I'm actually not 100% sure the best way to use UTM codes here, but I think examples would be good for the folks figuring it out, since I'm not confident.

File: content/pages/learning-hub/advertiser-guide.md [hunk 4]
```diff
@@ -100,13 +100,20 @@ but it's worth trying more focused variations as well.
...
-This keeps your measurements simple and lets you easily compare the actual conversions
-you're seeing from us against any other network where your ads are running.
-We are confident in our performance and think you'll see the results speak for themselves.
+we recommend adding UTM parameters to your landing page links.
+This keeps your measurements simple and lets you easily compare the conversions
+you're seeing from us against other networks you're using.
+We are confident in our performance and know the results will speak for themselves.
+
+We also have a few variables you can use in your landing page links
+in order to capture data about the publisher where the ad was clicked or the specific advertisement.
```
Comment (ericholscher) [inline content/pages/learning-hub/advertiser-guide.md:109]:
```suggestion
in order to capture data about the publisher where the ad was clicked or the specific advertisement:

* `${publisher}` is used to insert the name of the publisher
* `${advertisement}` is used to insert the name of the ad that was served

```

I think folks are likely gonna ask for `${flight}` here soon as well, if this gets used.

**Review by ericholscher — APPROVED** [review]
Looks great!

File: content/pages/learning-hub/advertiser-guide.md (hunk 4 above)
Comment (davidfischer) [reply content/pages/learning-hub/advertiser-guide.md:109]: That should be reasonably straight-forward to add if needed.

**Comment by ericholscher** [comment]
Had an advertiser _just_ ask about this, so gonna merge and send this to them 🚀

**Pull request merged**